core/src/main/scala/com/microsoft/azure/synapse/ml/core/contracts/Params.scala

```diff
@@ -205,3 +205,18 @@ trait HasGroupCol extends Params {
   /** @group getParam */
   def getGroupCol: String = $(groupCol)
 }
+
+trait HasExcludedFeatureCols extends Params {
+
+  /** The names of the hasExcludedCols
+    *
+    * @group param
+    */
+  val excludedFeatureCols = new StringArrayParam(this, "excludedFeatureCols", "The names of the columns to be excluded from featuring")
+
+  /** @group setParam */
+  def setExcludedFeatureCols(value: Array[String]): this.type = set(excludedFeatureCols, value)
+
+  /** @group getParam */
+  def getExcludedFeatureCols: Array[String] = $(excludedFeatureCols)
+}
```

Comment:
Simplify the naming.
```suggestion
trait HasExcludedFeatures extends Params {

  /** The names of the features to exclude
    *
    * @group param
    */
  val excludedFeatures = new StringArrayParam(this, "excludedFeatures", "The names of the features to be excluded from AutoTrainer")

  /** @group setParam */
  def setExcludedFeatures(value: Array[String]): this.type = set(excludedFeatures, value)

  /** @group getParam */
  def getExcludedFeatures: Array[String] = $(excludedFeatures)
}
```